src/lib.rs

```diff
@@ -72,6 +72,10 @@ pub extern "C" fn init(
                 1220296082861326378,
                 format!("{}/editor/nvchad.png", GITHUB_ASSETS_URL),
             ),
+            "astronvim" => (
+                1009122352916857003,
```

Comment:
```suggestion
                1230866983977746532,
```

Reply:
we do maintain a Discord Apps Team of our own for Cord, is it fine by you?

Reply:
Updated the commit!